package.json

```diff
@@ -15,6 +15,10 @@
     "@types/react-dom": "18.2.7",
     "eslint": "8.47.0",
     "eslint-config-next": "13.4.17",
+    "flowbite": "^1.8.1",
+    "flowbite-react": "^0.5.0",
+    "mobx": "^6.10.0",
```

Comment:
Can we try and avoid global state ? unless it is really needed and if so maybe we can use React Context. And if there are cases which this can't satisfy then we can add mobx at that point.

Reply:
`Flowbite` works with tailwind 
I would suggest we use `mobx`  for a better authenticate system. I want to avoid prop drilling as much as possible. It will helps us to write clean code.

Reply:
sounds good.